<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qseqeditframe64</class>
 <widget class="QFrame" name="qseqeditframe64">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="topBar1Layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="m_label_seqnumber">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>1024</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_entry_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Untitled</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_bpm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_bpm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_bw">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>W</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_bw">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_length">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>L</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_length">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_toggle_transpose">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_chord">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Ch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_chord">
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_bus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Buss</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_bus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_channel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_channel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>831</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="topBar2Layout">
    <item>
     <widget class="QPushButton" name="m_button_undo">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_redo">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_quantize">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Q</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_tools">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_toggle_follow">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_snap">
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>S</string>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>12</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Ctrl+S</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_snap">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_note">
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_note">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_zoom">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_zoom">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_key">
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>K</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_key">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_scale">
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_scale">
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_sequence">
      <property name="maximumSize">
       <size>
        <width>36</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Seq</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_sequence">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="timeScrollArea">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>80</y>
     <width>761</width>
     <height>30</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>759</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="keysScrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>41</width>
     <height>231</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>39</width>
      <height>229</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="qscrollmaster" name="rollScrollArea" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>761</width>
     <height>241</height>
    </rect>
   </property>
   <property name="widgetResizable" stdset="0">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3" native="true">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>10</y>
      <width>741</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="eventScrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>770</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>768</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="dataScrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>377</y>
     <width>771</width>
     <height>121</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>128</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>769</width>
      <height>119</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>811</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bottomBarLayout">
    <item>
     <widget class="QPushButton" name="m_button_event">
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Event</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_data">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_combo_data">
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_lfo">
      <property name="maximumSize">
       <size>
        <width>48</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>LFO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_toggle_play">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_toggle_thru">
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Thru</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>QRecord</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_rec_type">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Merge</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_button_rec_vol">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Fixed 127</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qscrollmaster</class>
   <extends>QWidget</extends>
   <header>qscrollmaster.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
